ufl/finiteelement/hdivcurl.py

```diff
@@ -86,3 +86,31 @@ def __str__(self):
     def shortstr(self):
         "Format as string for pretty printing."
         return "HCurlElement(%s)" % str(self._element.shortstr())
+
+
+class WithMapping(FiniteElementBase):
+    def __init__(self, wrapee, mapping):
```

Comment:
Can you add a docstring here describing what this does?

Reply:
done